src/lib.rs

```diff
@@ -86,18 +86,10 @@ impl<T> Gc<T> {
     pub fn alloc_blank(l: Layout) -> *mut T {
         let (layout, uoff) = Layout::new::<usize>().extend(l).unwrap();
         // In order for our storage scheme to work, it's necessary that `uoff -
-        // sizeof::<usize>()` gives a valid alignment for a `usize`. There are
-        // only two cases we need to consider here:
-        //   1) `object`'s alignment is smaller than or equal to `usize`. If so,
-        //      no padding will be added, at which point by definition `uoff -
-        //      sizeof::<usize>()` will be exactly equivalent to the start point
-        //      of the layout.
-        //   2) `object`'s alignment is bigger than `usize`. Since alignment
-        //      must be a power of two, that means that we must by definition be
-        //      adding at least one exact multiple of `usize` bytes of padding.
-        //      The assert below is thus paranoia writ large: it could only
```

Comment:
I only meant you should delete the sentence "the assert below". Case (2) is still important for you, but you just don't have the `assert`!

Reply:
Oh I see. Sorry, fixed that.